example/glue/configure_includes/configure_common.mk

```diff
@@ -24,5 +24,6 @@ CONFIGURE_ARGS += \
   --enable-OMR_OMRSIG \
   --enable-fvtest \
   --enable-OMR_GC_SEGREGATED_HEAP \
+  --enable-OMR_GC_MODRON_CONCURRENT_MARK \
```

Comment:
I would prefer if we disabled this by default until the testing has been updated.  This will also ensure that all new glue layer APIs are properly if defined